hutch_python/startup_script.py

```diff
@@ -0,0 +1,23 @@
+"""
+Initialize the ELogPoster callback and subscribe it to the RunEngine
+
+Startup script files to be run after ipython is initialized via:
+``c.InteractiveTerminalApp.exec_files``
+
+Code here will take executed after both IPython has been loaded and
+the namespace has been populated with hutch objects.
+
+These will be run as standalone python files, and should not be
+imported from.
+"""
+
+import IPython
+from nabs.callbacks import ELogPoster
```

Comment:
We need to update our `nabs` dependency, the `ELogPoster` is in `v1.3.0`
https://github.com/pcdshub/hutch-python/blob/5f77e01ee28f9f59446aea738215c862c63a6e3d/conda-recipe/meta.yaml#L31

Reply:
@tangkong did this make it in?

Reply:
Ah I missed it, I'll do that real quick